pkg/storage/blockdev.go

```diff
@@ -312,6 +317,59 @@ func tryVFAT(devname string) (uuid string) {
 	return uuid
 }
 
+const (
```

Comment:
these try functions all look so similar. Can you merge them into one?
Also, could they return 
(string, error) and could you have them not print.

I realize you're inheriting a problem here so I'm approving it but this could use some rethinking. Bloated code has a real cost in FLASH space.